openwisp_monitoring/device/static/monitoring/css/leaflet.fullscreen.css

```
@@ -1,5 +1,5 @@
 .njg-container .leaflet-control-fullscreen a {
-  background: #fff url(fullscreen.png) no-repeat 0 0;
+  background:var(--ow-color-white) url(fullscreen.png) no-repeat 0 0;
```

Stylelint (17.4.0)

[error] Expected quotes around "url" function argument (function-url-quotes)

(function-url-quotes)